src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -425,7 +422,7 @@ internal override ImmutableArray<NamedTypeSymbol> GetInterfacesToEmit()
             return InterfacesNoUseSiteDiagnostics();
         }
 
-        internal override NamedTypeSymbol GetDeclaredBaseType(ConsList<Symbol> basesBeingResolved)
+        internal override NamedTypeSymbol GetDeclaredBaseType(ConsList<Symbol> basesBeingResolved, bool ignoreNonNullTypesAttribute = false)
         {
             return GetDeclaredBaseType(ignoreNullability: false);
```

Comment:
>ignoreNullability: false [](start = 39, length = 24)

Should this be using the constructor parameter? If not, it would be good to put reasoning in a comment.

Reply:
I'll add a PROTOTYPE comment, as the PE scenarios will be addressed in follow-up scenarios. 
My initial thought is that the two flags have different meanings, but it's good to double-check.

---
In reply to: [200710926](https://github.com/dotnet/roslyn/pull/28320#discussion_r200710926) [](ancestors = 200710926)